src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -250,14 +255,24 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
-    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
-    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj" />
-    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
+    <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj">
```

Comment:
@joperezr  or @weshaggard should comment on this. I thought our build was set up to "do the right thing". I don't see other cases where projects set this manually.